crates/holochain_sqlite/src/sql/cell/update_dep_create_link.sql

```diff
@@ -11,11 +11,14 @@ WHERE
     SELECT
       1
     FROM
-      DhtOp AS OP_DEP
+      DhtOp AS DhtOpDep
     WHERE
-      OP_DEP.action_hash = DhtOp.dependency
-      AND OP_DEP.when_integrated IS NOT NULL
-      AND OP_DEP.type = :create_link
+      (
+        DhtOpDep.action_hash = DhtOp.dependency
```

Comment:
Is it possible to have the convention to store chain op deps in `DhtOp.dependency` and warrant op dependency 2 in `DhtOp.dependency2`? This change wouldn't be needed then.

Reply:
I'm not sure what you mean. The way I read this, it sounds like you're suggesting putting a chain op dep in one slot, and one of the warrant op deps in the other?

Reply:
Sorry, yes, exactly. Keep the first slot `dependency` for chain ops' sys dependency or a warrant op's first dependency accordingly, and the second slot only for the warrant op's second dependency.

Reply:
Oh yes of course. As you say it is how it is. And I see that the comment you're making is that all of these queries are only for chain ops, which have no second dependency, and so including that is redundant, which is also correct. I'll change that back to only check the first dep.